EnhancedMediaPlayer/MediaPlayer.swift

```diff
@@ -18,6 +18,16 @@ public struct MediaPlayer: UIViewControllerRepresentable {
     }
 
     public func updateUIViewController(_ mediaPlayerController: MediaPlayerViewController, context: Context) {
```

Comment:
Can't you annotate this with `@MainActor` instead?

Reply:
I made this particular implementation with @AmadeusK525 . Do you remember if there was any special reason to create the `extension`, or if it was just to make code smaller in `updateUIViewController`?

Reply:
I tired marking the `updateUIViewConroller` method as `@MainActor` but I kept getting the warning. I removed the `MainActor` extension to make the code familiar to readers